requirements.txt

```diff
@@ -4,6 +4,7 @@ BeautifulSoup4
 pyyaml
 lxml
 requests>=2.5.3
+certifi>=2015.04.28
```

Comment:
Yeah, unusual. The purpose of the library is just to repackage Mozilla/NSS's root store as a PEM file, so it works. I noticed the scheme used to be YY.MM.DD previously.